<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicDialog</class>
 <widget class="QDialog" name="GraphicDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2250</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="MFailsGraphic" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>m</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>691</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addOnFGPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>740</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить в КСА</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restartPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>740</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Инициализировать сначала</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="LamdaGraphic" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>λ</string>
    </property>
   </widget>
  </widget>
  <widget class="QCustomPlot" name="PGraphic" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>400</y>
     <width>681</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>t</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>410</y>
     <width>16</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="numFailsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общее колличество отказов:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="numFailsLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>addOnFGPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GraphicDialog</receiver>
   <slot>add_FG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>747</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>782</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restartPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GraphicDialog</receiver>
   <slot>atFirstFG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>784</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_FG()</slot>
  <slot>atFirstFG()</slot>
 </slots>
</ui>
